thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js

```diff
@@ -124,43 +125,11 @@ const processRangeParams = (bucketUnit, duration, start, end) => {
   return { startStamp, endStamp, baseStart, baseEnd };
 };
 
-/**
- * Builds the graph metric URL from config settings
- * TODO: Pull this into utils if used by create-alert
- * @param {Object} cfg - settings for current metric graph
- * @param {String} maxTime - an 'bookend' for this metric's graphable data
- * @return {String} URL for graph metric API
- */
-const buildGraphConfig = (config, maxTime) => {
-  const dimension = config.exploreDimensions ? config.exploreDimensions.split(',')[0] : 'All';
-  const currentEnd = moment(maxTime).isValid()
-    ? moment(maxTime).valueOf()
-    : buildDateEod(1, 'day').valueOf();
-  const formattedFilters = JSON.stringify(parseProps(config.filters));
-  // Load less data if granularity is 'minutes'
-  const isMinutely = config.bucketUnit.toLowerCase().includes('minute');
-  const duration = isMinutely ? { unit: 2, size: 'week' } : { unit: 1, size: 'month' };
-  const currentStart = moment(currentEnd).subtract(duration.unit, duration.size).valueOf();
-  const baselineStart = moment(currentStart).subtract(1, 'week').valueOf();
-  const baselineEnd = moment(currentEnd).subtract(1, 'week');
-  // Prepare call for metric graph data
-  const metricDataUrl =  `/timeseries/compare/${config.id}/${currentStart}/${currentEnd}/` +
-    `${baselineStart}/${baselineEnd}?dimension=${dimension}&granularity=` +
-    `${config.bucketSize + '_' + config.bucketUnit}&filters=${encodeURIComponent(formattedFilters)}`;
-  // Prepare call for dimension graph data
-  const topDimensionsUrl = `/rootcause/query?framework=relatedDimensions&anomalyStart=${currentStart}` +
-    `&anomalyEnd=${currentEnd}&baselineStart=${baselineStart}&baselineEnd=${baselineEnd}` +
-    `&analysisStart=${currentStart}&analysisEnd=${currentEnd}&urns=thirdeye:metric:${config.id}` +
-    `&filters=${encodeURIComponent(formattedFilters)}`;
-
-  return { metricDataUrl, topDimensionsUrl };
-};
-
 /**
  * Setup for query param behavior
  */
 const queryParamsConfig = {
-  refreshModel: true,
+  //refreshModel: true,
```

Comment:
Do we need this here, or can we delete it?

Reply:
Yes, decided not to use refreshModel